telco-ran/configuration/source-crs/node-tuning-operator/TunedPowerCustom.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: tuned.openshift.io/v1
+kind: Tuned
+metadata:
+  name: power-saving-performance-patch
+  namespace: openshift-cluster-node-tuning-operator
+  annotations:
+    ran.openshift.io/ztp-deploy-wave: "10"
+spec:
+  profile:
+    - name: power-saving-performance-patch
+      # Please note:
+      # - The 'include' line must match the TuneD profile to include and the priority must be lower than the priority of the profile it includes.
+      data: |
+        [main]
+        summary=Custom power saving configuration changes profile inherited from ran-du-performance
+        include=ran-du-performance
+        [variables]
+        # Note: Edit the above max_percent and scaling_governor variables with consultation of the following links for more information and applicable values:
+        # https://docs.redhat.com/en/documentation/openshift_container_platform/4.19/html/edge_computing/managing-cluster-policies-with-policygentemplate-resources#ztp-using-pgt-to-maximize-power-savings-mode_ztp-advanced-policy-config
+        # https://www.kernel.org/doc/Documentation/cpu-freq/governors.txt
+        max_percent=$max_perf_pct
+        scaling_governor=$governor
+        [bootloader]
+        cmdline_add=cpufreq.default_governor=${scaling_governor}
+        [sysfs]
+        /sys/devices/system/cpu/intel_pstate/max_perf_pct=${max_percent}
```

Comment:
Overriding the max_perf_pct is one way to cap the frequency, but there are others. I'm just wondering what a customer would do if they don't want to do it this way? Would they create their own power-saving-performance-patch?

Reply:
They certainly can. This is being provided as an example for two reasons:
- It's the only way we tell them to do it today, so we are preserving that (it matches the downstream docs, too)
- To exemplify the general mechanism of "How do I add a tuned performance profile override and still maintain the Red Hat-provided RAN DU settings

We can make the docs more clear that this is a general override mechanism and they can add other/alternate settings in the same way.